api/swagger/server/restapi/configure_thyra_server.go

```diff
@@ -84,9 +102,10 @@ func configureTLS(tlsConfig *tls.Config) {
 		var err error
 
 		tlsConfig.Certificates = make([]tls.Certificate, 1)
-		tlsConfig.Certificates[0], err = tls.X509KeyPair(unsecureCertificate, unsecureKey)
-
+		certiff, err := tls.X509KeyPair(unsecureCertificate, unsecureKey)
+		tlsConfig.Certificates = append(tlsConfig.Certificates, certiff)
 		if err != nil {
+			fmt.Println("error: unable to load certificate and key files.")
```

Comment:
I think you should use `log` instead of
```suggestion
			log.Println("error: unable to load certificate and key files.")
```